French/e107_themes/bootstrap3-troli/languages/French_admin.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
bootstrap3-troli is not a core theme.  It's best to export a language-pack from a fresh install of e107 v.2.1.4, rather than an upgrade or some installation that contains third-party elements. All the rest looks great, thank you,  if not for these additional files.